core/eval.go

```diff
@@ -1792,3 +1792,45 @@ func evalPersist(args []string) []byte {
 
 	return RESP_ONE
 }
+
+// evalEXPIREAT sets the expiration time for a specified key.
+// The expiration time is set based on the current time plus the provided seconds.
+// Arguments:
+//
+//	args[0] - The key for which to set the expiration time.
+//	args[1] - The number of seconds from the current time after which the key should expire.
+//
+// Returns:
+//   - Encoded OK response if the operation is successful.
+//   - Encoded error response if the number of arguments is incorrect, if the expiration time is invalid, or if the key does not exist.
+func evalEXPIREAT(args []string) []byte {
+	if len(args) != 2 {
+		return Encode(errors.New("ERR wrong number of arguments for 'EXPIREAT' command"), false)
+	}
+
+	key := args[0]
+	seconds, err := strconv.ParseInt(args[1], 10, 64)
+	if err != nil {
+		return Encode(errors.New("ERR invalid expire time"), false)
```

Comment:
Please use the same error message as redis - `ERR value is not an integer or out of range`

Reply:
Please resolve this.

Reply:
The error message still needs to be fixed